test/helpers/fixture-helper.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const callsites = require('callsites');
+
```

Comment:
This is really wet, could you make a single method that took some variables and worked in both instances?

```javascript
function __fixturePathFor(x, y, z) {
  // common code
}

function __htmlFixturePath(path) {
  return __fixturePathFor('html', path);
}

function __jsonFixturePath(path) {
  return __fixturePathFor('json', path);
}
```